Update storage-service-encryption.md

AWahlqvist opened the pull request:
I'm guessing this FAQ should be updated since the encryption settings for storage accounts in the Azure Portal states:
"Please note that after enabling Storage Service Encryption, only new data will be encrypted, and any existing files in this storage account will retroactively get encrypted by a background encryption process."

azurecla commented:
Hi __@NotThatSmith__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
@NotThatSmith - Thanks for the update - this is already fixed in the article.

AWahlqvist committed:
Update storage-service-encryption.md

I'm guessing this FAQ should be updated since the encryption settings for storage accounts in the Azure Portal states:
"Please note that after enabling Storage Service Encryption, only new data will be encrypted, and any existing files in this storage account will retroactively get encrypted by a background encryption process."
